dashboard/modules/job/job_manager.py

```diff
@@ -176,6 +176,16 @@ def __init__(
         # Windows Job Object used to handle stopping the child processes.
         self._win32_job_object = None
 
+        # Logger object to persist JobSupervisor logs in separate file.
```

Comment:
How about lets move JobSupervisor class to it's own file `dashboard/modules/job/job_supervisor.py` so that people won't accidentally use the `logger` for job manager.

Reply:
Good idea, I moved `JobLogStorageClient` to a separate file also because it is being used by `JobSupervisor` and `JobManager`